region/lowernorfair/east/Mickey Mouse Room.json

```diff
@@ -798,6 +866,499 @@
+            {"getBlueSpeed": {"usedTiles": 23, "openEnd": 2}},
+            {"heatFrames": 200}
+          ]},
+          {"and": [
+            "h_heatProof",
+            "canDodgeWhileShooting",
+            {"enemyKill": {
+              "enemies": [["Dessgeega"], ["Dessgeega"], ["Dessgeega"], ["Multiviola"], ["Multiviola"]],
+              "excludedWeapons": ["Bombs", "PseudoScrew", "PowerBomb", "PowerBeam"]}
+            }
+          ]},
+          {"and": [
+            {"heatFrames": 400},
+            "Plasma",
+            {"or": [
+              "Wave",
+              {"heatFrames": 100}
+            ]}
+          ]},
+          {"and": [
+            {"enemyKill": {
+              "enemies": [["Dessgeega"], ["Dessgeega"], ["Dessgeega"], ["Multiviola"], ["Multiviola"]],
+              "explicitWeapons": ["Missile"]}
+            },
+            {"heatFrames": 500}
+          ]},
+          {"and": [
+            {"enemyKill": {
+              "enemies": [["Dessgeega"], ["Dessgeega"], ["Dessgeega"], ["Multiviola"], ["Multiviola"]],
+              "explicitWeapons": ["Super"]}
+            },
+            {"heatFrames": 400}
+          ]},
+          {"and": [
+            "ScrewAttack",
+            {"heatFrames": 320}
+          ]}
+        ]},
+        {"or": [
+          {"resourceAvailable": [{"type": "ReserveEnergy", "count": 1}]},
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+          ]}
+        ]},
+        "h_shinechargeMaxRunway",
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt",
+        {"heatFrames": 30}
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Use normal bombs or a Power Bomb to open the bomb blocks, then navigate the Multiviola boxes.",
+        "Kill the two right Dessgeegas, then Crystal Flash. You cannot use Power Bombs or X-Ray after the",
+        "Crystal Flash until R-Mode ends. Clear the rest of the runway. Use the Multiviola at the end to",
+        "interrupt, or use a heat interrupt."
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "canBePatient",
+        {"or": [
+          {"and": [
+            "h_usePowerBomb",
+            {"heatFrames": 180}
+          ]},
+          {"and": [
+            {"heatFrames": 160},
+            {"or": [
+              "ScrewAttack",
+              {"and": [
+                "h_useMorphBombs",
+                {"heatFrames": 60}
+              ]}
+            ]}
+          ]}
+        ]},
+        {"or": [
+          {"and": [
+            {"heatFrames": 400},
+            {"enemyDamage": {"enemy": "Multiviola", "type": "contact", "hits": 3}}
```

Comment:
The full energy requirement in `h_RModeCanRefillReserves` makes it so you can't tank the damage.

Reply:
It can still work on the `disable: ETank` path as long as you still have 99 energy.